kubernetes/samples/scripts/terraform/oke.create.sh

```diff
@@ -68,6 +69,60 @@ function createCluster () {
     terraform apply -auto-approve -var-file=${terraformVarDir}/${clusterTFVarsFile}.tfvars
 }
 
+function createRoleBindings () {
+kubectl -n kube-system create serviceaccount $okeclustername-sa
+kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:$okeclustername-sa
+TOKENNAME=`kubectl -n kube-system get serviceaccount/$okeclustername-sa -o jsonpath='{.secrets[0].name}'`
+TOKEN=`kubectl -n kube-system get secret $TOKENNAME -o jsonpath='{.data.token}'| base64 --decode`
+kubectl config set-credentials $okeclustername-sa --token=$TOKEN
+kubectl config set-context --current --user=$okeclustername-sa
+}
+
+function checkClusterRunning () {
+
+echo 'Confirm we have kubectl working...'
```

Comment:
add some indentation for the function body.

Reply:
fixed